.github/workflows/build_image.yaml

```
@@ -19,7 +19,7 @@ env:
   TIMESCALE_STATIC_PRIMARY: "true"
   TIMESCALE_TSDB_ADMIN: "1.0.0"
   TIMESCALEDB_TOOLKIT_EXTENSIONS: 1.6.0 1.7.0 1.8.0 1.10.1 1.11.0 1.12.0 1.12.1 1.13.0
-  TIMESCALE_OSM_EXTENSION: 0.0.2 0.0.3
+  TIMESCALE_OSM_EXTENSION: 0.0.4 0.0.5
```

```suggestion
  TIMESCALE_OSM_EXTENSION: 0.0.5
```

The OSM extension doesn't install versioned files - it ends up as `/usr/lib/postgresql/<version>/lib/timescaledb_osm.so`, so there's no need to have multiple versions here.

---

@graveland thanks, updated.